packages/docs-page/style.css

```diff
@@ -1,160 +0,0 @@
-/** @TODO: remove once search is converted to use css modules https://app.asana.com/0/1100423001970639/1199982445969979/f **/
-@import '~@hashicorp/react-search/style.css';
-
-@custom-media --mobile-viewports (max-width: 939px);
-@custom-media --desktop-viewports (min-width: 940px);
-
-/* @TODO: adjustment for versioned docs */
-@media (max-width: 767px) {
-  .g-alert-banner {
-    display: none !important;
-  }
-}
```

Comment:
Note: I had to drop these styles entirely as they're no longer possible in CSS modules.

I considered also trying to convert `alert-banner` as part of this PR, but didn't want to balloon the scope of this PR - it's really meant to just support global style changes, and I think it's important to keep the focus there. I'm planning instead to include this style globally in consuming projects that use `docs-page` (ie, all `.io` sites), with an eye on removing that tiny addition once we've decided on a longer term solution.

Task to discuss: https://app.asana.com/0/1100423001970639/1200634120002244/f

Reply:
Updated: rolled in the `hideOnMobile` prop approach into this PR.